### PR TITLE
fix: prevent firing `exitFullscreen` event on all maps of the page in…

### LINKED ISSUE #117: Button state incorrect when multiple maps on a page
If there are multiple maps on a page, e.g. showing different walking routes in an area, if you enter and then exit fullscreen on one map, the fullscreen button on all the other maps then shows the exit fullscreen icon and text, even though they are clearly not fullscreened.

This only affects buttons which have not yet been clicked. If I then fullscreen/exit on a different map, the first map to be fullscreened shows the correct icon state.

### PR DESCRIPTION
**Pull request by brunob**
… `_handleFullscreenChange()`

Fix #117

### CODE REVIEW

**Comment by brunob** [comment]
Any thought @BePo65 ?

**Comment by brunob** [comment]
We should also put `map.invalidateSize();` in `if (ev.target === map.getContainer()` test for sanity.

**Comment by BePo65** [comment]
The solution is pretty simple. but give me a few days to reproduce the effect and the solution. I think that I can give you an evaluation until this weekend.

File: Control.FullScreen.js [hunk 1]
```diff
@@ -240,10 +240,10 @@
...
 
-		_handleFullscreenChange: function () {
+		_handleFullscreenChange: function (ev) {
 			var map = this._map;
 			map.invalidateSize();
-			if (!this._screenfull.isFullscreen && !map._exitFired) {
+			if (ev.target === map.getContainer() && !this._screenfull.isFullscreen && !map._exitFired) {
 				map.fire('exitFullscreen');
 				map._exitFired = true;
 				map._isFullscreen = false;
```
Comment (BePo65) [inline Control.FullScreen.js:249]: I think, this is the line after which `map.invalidateSize();` should be added (see [comment ](https://github.com/brunob/leaflet.fullscreen/pull/118#issuecomment-1881796644)from brunob)

**Comment by BePo65** [comment]
So now I created a demo (on my pc only :-) and can reproduce the error and the solution. IMHO this is a fine solution that should be added to this project. 👍

**Pull request merged**